src/citrine/resources/data_concepts.py

```diff
@@ -27,7 +29,23 @@
 CITRINE_TAG_PREFIX = 'citr_auto'
 
 
-class DataConcepts(DictSerializable, PolymorphicSerializable['DataConcepts'], ABC):
+class DataConceptsMeta(DictSerializableMeta):
+    """Data Concepts metaclass for handling serialization."""
+
+    def __init__(cls, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        resolved = next((b.typ for b in cls.__bases__ if getattr(b, "typ", None) is not None),
+                        None)
+        if resolved is not None:
+            cls._typ_stash = resolved
+        cls.typ = String("type")
```

Comment:
We have to do weird things here because GEMD.typ is a class-value, rather than an object value, property, or classmethod.  Ideally, it'd be a classmethod+property, but Python 3.12 dropped support for that concept.